mfes/content/public/content/preview/coreplugins/org.ekstep.videorenderer-1.1/renderer/plugin.js

```
@@ -0,0 +1,385 @@
+                    hls: {
+                        overrideNative: true,
+                    }
+                }
+            }, function () {
+                this.on('downloadvideo', function () {
+                    EkstepRendererAPI.dispatchEvent("renderer:splash:hide");
+                    console.log("downloadvideo");
+                })
+            });
+        } else {
+            var videoPlayer = videojs('videoElement', {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'videoPlayer'. Consider to delete it or rename it.

'videoPlayer' is defined here:


(lint/suspicious/noRedeclare)